server/routes/journeys/develop-an-initial-plan/intervention-details/view.njk

```diff
@@ -0,0 +1,75 @@
+{% from "moj/components/date-picker/macro.njk" import mojDatePicker %}
+{% from "govuk/components/character-count/macro.njk" import govukCharacterCount %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% extends "partials/layout.njk" %}
+
+{% set pageTitle = " Intervention details - Develop an initial plan" %}
+
+{% block innerContent %}
+    <span class="govuk-caption-l">Develop an initial plan</span>
+    <h1 class="govuk-heading-l govuk-!-margin-bottom-7">Intervention details</h1>
+    <form method="post">
+        <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+
+        <h2 class="govuk-body govuk-!-font-weight-bold">Identified need summary</h2>
```

Comment:
body font and heading font are different. i think heading font is expected here?